students/rodicaf/src/fibonacci.ts

```diff
@@ -0,0 +1,31 @@
+export const fibrec  = (n:number) : number => {
```

Comment:
it us better to use defensive programming. What does happen when number is negative?